src/EnrollChoicesController.js

```diff
@@ -200,6 +201,11 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
       }
 
       this.state.set('pageType', pageType);
+
+      var transaction = this.options.appState.get('transaction');
```

Comment:
looking at this again, I don't think this is the best way to do this. We shouldn't be getting the `transaction` here, that is kind of abstract for when we start and manage one.
I think we should put this inside `AppState`, similar to how we have [prevLink](https://github.com/okta/okta-signin-widget/blob/master/src/models/AppState.js#L430)

Reply:
ahh makes sense 👍